ocf_datapipes/training/common.py

```diff
@@ -37,6 +37,32 @@
 logger = logging.getLogger(__name__)
 
 
+def is_config_and_path_valid(use_flag: bool, config, filepath_resolver) -> bool:
```

Comment:
Not blocking but could add more type hints here for the rest of the parameters, I believe config here is the config input data which is an [InputData type]( https://github.com/openclimatefix/ocf_datapipes/blob/main/ocf_datapipes/config/model.py#L729)